pkgs/development/php-packages/phpcs/default.nix

```diff
@@ -1,35 +1,27 @@
-{ mkDerivation, fetchurl, makeWrapper, lib, php }:
+{ stdenv
```

Comment:
File can be removed, it is now dead code.

Reply:
Fixed.